### PR TITLE
f/initial-version

### PR DESCRIPTION
**Pull request by justinyost**
Initial Working Version of this plugin.

### CODE REVIEW

**Comment by justinyost** [comment]
This needs to be merged to then be published for Packagist.

File: tests/bootstrap.php [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+		'duration' => '+10 seconds'
+	]
+];
+
+Cake\Cache\Cache::config($cache);
+Cake\Core\Configure::write('Session', [
+	'defaults' => 'php'
+]);
+
+Cake\Core\Plugin::load('Crud', ['path' => ROOT . DS, 'autoload' => true]);
```
Comment (beporter) [inline tests/bootstrap.php:74]: Is this a copy/paste leftover, or do we really need the Crud plugin for testing?

File: config/routes.php [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?php
+use Cake\Routing\Router;
+
+Router::plugin('CreatorModifier', function ($routes) {
+	$routes->fallbacks('InflectedRoute');
+});
```
Comment (beporter) [inline config/routes.php:6]: Do we need to define routes for a behavior-only plugin? Is this required for something I'm not thinking of?

File: src/Controller/AppController.php [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace CreatorModifier\Controller;
+
+use App\Controller\AppController as BaseController;
+
+class AppController extends BaseController {
+}
```
Comment (beporter) [inline src/Controller/AppController.php:8]: Same question: Is having an AppController in a plugin mandatory? Can this be removed?

**Comment by beporter** [comment]
This has to have been about the easiest plugin release ever, huh?

**Comment by beporter** [comment]
You know what, I want to get this out so I can keep moving on the rest. We can address the questions above in a follow-up PR if necessary. Nice work Justin. :+1:

**Pull request merged**